src/merkle_reg.rs

```diff
@@ -1,8 +1,8 @@
 use core::convert::Infallible;
 use core::fmt;
-use std::collections::{BTreeMap, BTreeSet};
-
 use quickcheck::{Arbitrary, Gen};
+use serde::{Deserialize, Serialize};
+use std::collections::{BTreeMap, BTreeSet};
```

Comment:
please keep the std/core imports up in the first group of imports, third party imports in the second group and local crate imports in the third group